plenum/server/consensus/msg_validator.py

```diff
@@ -94,9 +94,9 @@ def validate(self, msg):
 
         # 4. Check if from future view
         if view_no > self._data.view_no:
-            return STASH_VIEW, FUTURE_VIEW
+            return STASH_WAITING_NEW_VIEW, FUTURE_VIEW
```

Comment:
I would use `STASH_VIEW` instead of `STASH_WAITING_NEW_VIEW` in all places and remove `STASH_WAITING_NEW_VIEW` at all.

Reply:
Changed